arrow-docs/docs/arrow/typeclasses/traverse/README.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: docs-core
+title: Traverse
+permalink: /arrow/typeclasses/traverse/
+---
+
+## Traverse
+
+
+
+
+The `Traverse` typeclass is allowing to commute types from `F<G<A>>` to `G<F<A>>` over sequential execution of code.
+Traversal over a structure with an effect.
```

Comment:
`The main use of this is traversal over...`

Reply:
Thanks! See 4e5f8d68